rpc/sigpipe.go

```diff
@@ -0,0 +1,9 @@
+// +build !darwin
+
+package rpc
+
+import "net"
+
+func DisableSigPipe(c net.Conn) error {
+	return nil
```

Comment:
> Although this makes it so that we need to detect whether this is supported on the clients of this library, which makes that code a lot more verbose. With this just doing nothing all that is simpler.

If you want to avoid that, I'd rename the option something like `disableSigPipeOnDarwinOnly` or something like that, to avoid confusion.  Seems like a cop-out though.  Seems like something the caller _should_ be doing deliberately.  Presumably the option is being set from some place that somehow knows it's mobile already, so figuring out that it's darwin too in that place doesn't seem like it would add much complexity.